docs/tutorials/examples.rst

```diff
@@ -290,128 +290,10 @@ zipcode for each user:
-^^^^^^^^^^^^^^^^^^
-
-First you must ensure that:
-
-1. Confluent Schema Registry is up and running.
-2. ``ksql.schema.registry.url`` is set correctly in KSQL (see previous
-   section).
-
-Then you can use ``CREATE STREAM`` and ``CREATE TABLE`` statements to
-read from Kafka topics with Avro-formatted data and ``CREATE STREAM AS``
-and ``CREATE TABLE AS`` statements to write Avro-formatted data into
-Kafka topics.
-
-Example: Create a new stream ``pageviews`` by reading from a Kafka topic
-with Avro-formatted messages.
-
-.. code:: sql
-
-    CREATE STREAM pageviews
-      WITH (KAFKA_TOPIC='pageviews-avro-topic',
-            VALUE_FORMAT='AVRO');
-
-Example: Create a new table ``users`` by reading from a Kafka topic with
-Avro-formatted messages.
-
-.. code:: sql
-
-    CREATE TABLE users
-      WITH (KAFKA_TOPIC='users-avro-topic',
-            VALUE_FORMAT='AVRO',
-            KEY='userid');
-
-Note how in the above example you don’t need to define any columns or
-data types in the CREATE statement because KSQL will automatically infer
-this information from the latest registered Avro schema for topic
-``pageviews-avro-topic`` (i.e., the latest schema at the time the
-statement is first executed).
-
-If you want to create a STREAM or TABLE with only a subset of all the
-available fields in the Avro schema, then you must explicitly define the
-columns and data types.
-
-Example: Create a new stream ``pageviews_reduced``, similar to the
-previous example, but with only a few of all the available fields in the
-Avro data (here, only the two columns ``viewtime`` and ``pageid`` are
-picked).
-
-.. code:: sql
-
-    CREATE STREAM pageviews_reduced (viewtime BIGINT, pageid VARCHAR)
-      WITH (KAFKA_TOPIC='pageviews-avro-topic',
-            VALUE_FORMAT='AVRO');
-
-KSQL allows you to work with streams and tables regardless of their
-underlying data format. This means that you can easily mix and match
-streams and tables with different data formats (e.g. join a stream
-backed by Avro data with a table backed by JSON data) and also convert
-easily between data formats.
-
-Example: Convert a JSON stream into an Avro stream.
-
-.. code:: sql
-
-    CREATE STREAM pageviews_json (viewtime BIGINT, userid VARCHAR, pageid VARCHAR)
-      WITH (KAFKA_TOPIC='pageviews-json-topic', VALUE_FORMAT='JSON');
-
-    CREATE STREAM pageviews_avro
-      WITH (VALUE_FORMAT = 'AVRO') AS
-      SELECT * FROM pageviews_json;
-
-Note how you only need to set ``VALUE_FORMAT`` to Avro to achieve the
-data conversion. Also, KSQL will automatically generate an appropriate
-Avro schema for the new ``pageviews_avro`` stream, and it will also
-register the schema with Confluent Schema Registry.
-
 Running KSQL
 ------------
 
-KSQL supports various :ref:`modes of operation <modes-of-operation>`, including a standalone mode and a client-server mode.
+KSQL supports client mode and server mode.
 
 Additionally, you can also instruct KSQL to execute a single statement
```

Comment:
I think this section needs updating because IIRC the syntax for `ksql http://localhost:8088 --exec "SELECT * FROM pageviews LIMIT 5;"` has changed.  @dguy should know more because he was already asked by @ybyzek about this.

Reply:
Unfortunately, `exec` option was removed.  Now you can do this

```bin/ksql http://localhost:8080 <<EOF
> run script 'ksql.commands;
> exit
> EOF```

Reply:
Ok, here's a suggested for this paragraph with updated examples
----
Apart from using the KSQL CLI or launching KSQL servers with the `--queries-file` configuration, you can also execute KSQL statements from the command line.

Example 1:

```
$ ksql http://localhost:8088 <<EOF
> SHOW TOPICS;
> exit
> EOF
```
Example 2:

```
$ ksql http://localhost:8088 <<EOF
> RUN SCRIPT '/path/to/application.sql';
> exit
> EOF
```